hack/build/config.sh

```diff
@@ -22,6 +22,7 @@ VERBOSITY=${VERBOSITY:-1}
 PULL_POLICY=${PULL_POLICY:-Always}
 AAQ_NAMESPACE=${AAQ_NAMESPACE:-aaq}
 CR_NAME=${CR_NAME:-aaq}
+BUILD_ARCHES="amd64 arm64 s390x"
```

Comment:
```suggestion
BUILD_ARCHES="${BUILD_ARCHES:-amd64 arm64 s390x}"
```
This should be configurable.

Reply:
it is!